front50-cassandra/src/main/groovy/com/netflix/spinnaker/front50/config/CassandraConfig.groovy

```diff
@@ -28,8 +28,9 @@ import org.springframework.context.annotation.Configuration
 import org.springframework.core.env.Environment
 
 @Configuration
-@ConditionalOnExpression('${cassandra.enabled:true}')
+@ConditionalOnExpression('${spinnaker.cassandra.enabled:false}')
```

Comment:
I believe we were using `cassandra.enabled` previously as there are some `kork-cassandra` components that use that key to enable/disable themselves.